packages/block-library/src/latest-posts/index.php

```diff
@@ -21,6 +43,9 @@ function render_block_core_latest_posts( $attributes ) {
 		'suppress_filters' => false,
 	);
 
+	$block_core_latest_posts_excerpt = $attributes['excerptLength'];
+	add_filter( 'excerpt_length', 'get_block_core_latest_posts_excerpt', 999 );
```

Comment:
Can we locate this to somewhere in closer proximity to where we're expecting the filter to be applied? i.e. the `get_the_excerpt` function call.

Reply:
I want to set the filter outside of the foreach loop.